editor.js

```diff
@@ -82,7 +82,7 @@ function makeEditorShim(id) {
 
 
 function extension_editor_finish_run() {
-	if (typeof(unsafeWindow) !== 'undefined') {
+	if ((typeof(unsafeWindow) !== 'undefined') && (navigator.userAgent.indexOf('Mobile') === -1)) {
```

Comment:
do extensions even load on the editor page? i don't know if they do

On Thu, Dec 17, 2015 at 3:00 AM Wolvan <notifications@github.com> wrote:

> In editor.js
> <https://github.com/new-xkit/XKit/pull/873#discussion_r47878220>:
>
> > @@ -82,7 +82,7 @@ function makeEditorShim(id) {
> >
> >
> >  function extension_editor_finish_run() {
> > -	if (typeof(unsafeWindow) !== 'undefined') {
> > +	if ((typeof(unsafeWindow) !== 'undefined') && (navigator.userAgent.indexOf('Mobile') === -1)) {
>
> You know, all extensions get loaded before the editor does as far as I
> recall?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/873/files#r47878220>.
>